system/jlib/jtrace.cpp

```diff
@@ -59,6 +61,35 @@ namespace opentel_trace = opentelemetry::trace;
 
 using namespace ln_uid;
 
+
+class CustomOTELLogHandler : public opentelemetry::sdk::common::internal_log::LogHandler
+{
+    void Handle(opentelemetry::sdk::common::internal_log::LogLevel level,const char * file, int line, const char * msg,
+                const opentelemetry::sdk::common::AttributeMap &attributes) noexcept override
+    {
+        if (!isEmptyString(msg))
+        {
+            LogMsgCategory logCategory = MCdebugInfo;
```

Comment:
minor: Could have a const LogMsgCategory * and avoid the copy.  In this case the copy is not expensive, and code is not time critical, so doesn't need to change.

Reply:
this seems worthwhile